### DISCUSSION TITLE
Nimib IRL: starting the move to nimib-land

### DISCUSSION THREAD

**Discussion by pietroppeter** [pietroppeter/nimib#221] [Nimib speaking hours]
Today me and @HugoGranstrom are meeting IRL (In Real Life) for the first time since we started working on this project together! 🥳 

<img src="https://github.com/pietroppeter/nimib/assets/4997234/18571fd2-5b38-4ea3-9a34-1881bc406671" width="400">

We started to move a couple of repos to nimib-land organization:
- HugoGranstrom/nimiboost: this was very smooth and has very little impact
- pietroppeter/nblog: not as smooth as the other one. while GitHub repo links are redirect to the new organization, GitHub pages links are NOT redirected, so every nblog content linked is broken actually. The solution is probably to create again a pietroppeter/nblog repo and manually add redirects. If anyone knows of a better solution let us know.

If you are at PyCon Sweden and want to talk with us we will be there the next two days!

**Comment by Clonkk** [comment]
Awesome !
